conf/core/ConfigOptions.asciidoc

```diff
@@ -144,6 +144,34 @@ See also:
 
  * _Estimate Pairwise Relationships_, <<hootalgo>>
 
+=== changeset.buffer
+
+* Data Type: double
+* Default Value: `0.0`
+
+Value in degrees by which the specified AOI (convert.bounding.box) will be expanded when calculating
+changesets.  Setting this too large with some datasets may greatly increase the changeset
+derivation time.
+
+=== changeset.allow.deleting.from.features
+
+* Data Type: bool
+* Default Value: `true`
+
+If true, changesets derived can issue delete statements for the 'from' dataset (first dataset
+passed to the changeset deriver).  If false, no delete statements will be issued for the 'from'
+dataset.
+
+=== changeset.compare.status
+
+* Data Type: bool
+* Default Value: `false`
+
+If true, changeset elements are considered different if they have different statuses.  If false,
+then status does not matter during element comparison.  Setting this to true may result in many
+additional changeset modify statements when Hootenanny changes the status of elements to
+'conflated'.
```

Comment:
Yeah, this one is kind of up for debate at this point.  The problem I was seeing were quite a few reference features that the only change they had in them was changing hoot status from 1 to 3.  That qualifies as a change and resulted in a ton of additional modify statements, so I put an option in to keep that from happening....but may not be the best final solution.  Possibly the answer is to add a check in the part of the code that preserves the reference id's and if not write status=3 unless some tag or geometry actually changed, compared to the original reference feature.  I thought that's what it was doing from the beginning but apparently not.